src/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -0,0 +1,318 @@
+import { LightningElement, api, track, wire } from 'lwc';
+import { fireEvent } from 'c/pubsubNoPageRef';
+import { ShowToastEvent } from 'lightning/platformShowToastEvent';
+
+import { getNestedProperty } from 'c/utilCommon';
+import { getObjectInfo } from 'lightning/uiObjectInfoApi';
+import { getRecord} from 'lightning/uiRecordApi';
+
+import RECURRING_DONATION_INFO from '@salesforce/schema/npe03__Recurring_Donation__c';
+import ACCOUNT_OBJECT from '@salesforce/schema/Account';
+import CONTACT_OBJECT from '@salesforce/schema/Contact';
+
+import FIELD_NAME from '@salesforce/schema/npe03__Recurring_Donation__c.Name';
+import FIELD_ACCOUNT from '@salesforce/schema/npe03__Recurring_Donation__c.npe03__Organization__c';
+import FIELD_CONTACT from '@salesforce/schema/npe03__Recurring_Donation__c.npe03__Contact__c';
+import FIELD_DATE_ESTABLISHED from '@salesforce/schema/npe03__Recurring_Donation__c.npe03__Date_Established__c';
+import FIELD_STATUS from '@salesforce/schema/npe03__Recurring_Donation__c.Status__c';
+import FIELD_RECURRING_TYPE from '@salesforce/schema/npe03__Recurring_Donation__c.RecurringType__c';
+import FIELD_INSTALLMENT from '@salesforce/schema/npe03__Recurring_Donation__c.npe03__Installments__c';
```

Comment:
`FIELD_INSTALLMENT` -> `FIELD_PLANNED_INSTALLMENTS`?
Consequently, `fields.installment` -> `fields.plannedInstallments` in the `js` and `html` files.